solr/templates/statefulset.yaml

```diff
@@ -126,13 +126,33 @@ spec:
       containers:
         - name: solr
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+{{ if (eq .Values.solr_mode "local") }}
```

Comment:
I think that `command` and `args` should be able to be specified with something like:
```
{{ if .Values.command }}
          command:
{{ toYaml .Values.command | indent 12 }}
{{ end }}
{{ if .Values.args }}
         args:
{{ toYaml .Values.args | indent 12 }}
{{ end }}
```
The difference between the `EKS`, `GKE` and `local` clusters should then be specified from values files. We could keep them as comments in the `values.yaml` file if we want to document how to set the labels in solr for EKS and GKE.